src/plugins/mathcheck/README.md

```diff
@@ -26,31 +26,56 @@ Keynames are all either relative to to-be-tested key (starting with `./` or `../
 `check/math = "<= - @/testval1 * @/testval2 @/testval3"` tests if the keyvalue is less than or equal to testval1 - (testval2 * testval3) and yields an error if not.
 
 Full example:
-
-    kdb mount mathcheck.dump /example/mathcheck dump mathcheck
-    kdb set user/example/mathcheck/a 3.1
-    kdb set user/example/mathcheck/b 4.5
-    kdb set user/example/mathcheck/k 7.6
-    kdb setmeta user/example/mathcheck/k check/math "== + ../a ../b"
-    kdb set user/example/mathcheck/k 7.7   # fails
-
+```sh
+# Backup-and-Restore:/examples/mathcheck
+kdb mount mathcheck.dump /examples/mathcheck dump mathcheck
+kdb set /examples/mathcheck/a 3.1
+kdb set /examples/mathcheck/b 4.5
+kdb set /examples/mathcheck/k 7.6
+kdb setmeta user/examples/mathcheck/k check/math "== + ../a ../b"
+# should fail
```

Comment:
i'm not sure how to distinguish between # as a part of an command argument and # as the start of a command 
e.g. `ls #a` <- should this list the content of the current directory and treat `#a` as a comment or list the content of the directory `#a` ?